shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -22,15 +22,22 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
 import java.util.Map.Entry;
 
 /**
  * Utility for printing Alluxio configuration.
  */
 public final class GetConf {
-  private static final String USAGE = "USAGE: GetConf [--unit <arg>] [KEY]\n\n"
+  private static final String USAGE = "USAGE: GetConf [--unit <arg>] [--display-name] [-e] [KEY]"
```

Comment:
the translation of environment variables to property keys is specific to the Docker integration - outside the Docker integration we do not support configuring arbitrary property keys via environment variable. This makes it weird to add this `-e` flag. The `--display-name` flag is also weird from a user perspective - why would the caller need both the key and value if they just passed the key?

The exit code of `getConf` already says whether a key is valid or not, so I think we can get all the functionality we are looking for by calling the existing `getConf` on every translated environment variable and including it if the key is valid and the result is nonempty.

Reply:
Agree that the new `-e` and `--display-name` parameters are not very useful outside docker configuration. The main purpose for adding the environment variable translation to `getConf` is to make Java the single source of truth for property name resolution so we don't have to worry about it become out of sync whenever we make changes to the property names. I can remove these parameters from usage string to discourage people from using it. Let me know if you have any suggestions.

Reply:
I'm fine with using Java for the translation, but I don't think `getConf` is the right place to do it. Could we create a separate tool for translating both directions between environment variables and property keys? It should be clear that the environment variables only apply to the Docker integration.